docs/technical_reference/unit_models/ion_exchange_0D.rst

```diff
@@ -0,0 +1,317 @@
+
+        ions = m.fs.unit.config.property_package.ion_set
+
+**NOTE: The above example assumes you have already constructed a pyomo model named 'm' and attached an IDAES flowsheet named 'fs' to it.**
+
+Equations and Relationships
+---------------------------
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Separation factor", ":math:`\alpha = \frac{1}{La}`"
+   "Langmuir isotherm", ":math:`\alpha \frac{C_{b}}{C_{0}} (1-\frac{q_{eq}}{q_{max}}) = (1-\frac{C_{b}}{C_{0}})\frac{q_{eq}}{q_{max}}`"
+   "Reynolds number", ":math:`Re = \frac{u_{bed}d}{\mu}`"
+   "Schmidt number", ":math:`Sc = \frac{\mu}{D}`"
+   "Sherwood number", ":math:`Sh = \frac{1.09}{\epsilon}Re^{0.33}Sc^{0.33}`"
+   "Bed Peclet number", ":math:`Pe_{bed} = Pe_{p} \frac{Z}{d}`"
+   "Particle Peclet number", ":math:`Pe_{p} = 0.05 Re^{0.48}`"
+   "Resin capacity mass balance", ":math:`q_{max} = q_{avail} + q_{eq}`"
+   "Interstitial velocity", ":math:`u_{inter} = \frac{u_{bed}}{\epsilon}`"
+   "Resin surface area per vol", ":math:`a_{s} = 6 \frac{1-\epsilon}{d}`"
+   "Contact time", ":math:`t_{contact} = \frac{Z}{u_{inter}}`"
+   "Service flow rate", ":math:`SFR = \frac{Q_{p, in}}{V_{tot}}`"
+   "Flow through bed constraint", ":math:`\frac{Z \epsilon}{u_{bed}} = \frac{V_{bed} \epsilon}{Q_{p, in} / n_{op}}`"
+   "Total bed volume", ":math:`V_{tot} = V_{bed}n_{op}`"
+   "Column height", ":math:`H = Z + H_{distributor} + H_{underdrain} + H_{expan}`"
+   "Column volume calculated from bed volume", ":math:`V_{col} = H \frac{V_{bed}}{Z}`"
+   "Column volume calculated from column diameter", ":math:`V_{col} = \pi (\frac{D_{col}}{2})^{2} H`"
+   "Column diameter calculation", ":math:`(\frac{D_{col}}{2})^{2} = (\frac{H}{2X})^{2}`"
+   "Fluid mass transfer coeff", ":math:`k_{f} = \frac{D Sh}{d}`"
+   "Rate coefficient", ":math:`k = 6 \frac{(1-\epsilon)k_{f}}{\rho_{b}d}`"
+   "Height of transfer unit", ":math:`HTU = \frac{u_{bed}}{\rho_{b}k}`"
+   "Partition ratio", ":math:`\Lambda = \frac{q_{eq} \rho_{b}}{ñ_{in}}`"
+   "Left hand side of constant pattern solution", ":math:`lh = N(\tau - 1)`"
+   "Right hand side of constant pattern solution", ":math:`lh = 1 + \frac{\log{(C_{b}/C_{0})} - La \log{(1 - C_{b}/C_{0})}}{1 - La}`"
+   "Dimensionless time", ":math:`\tau = (\frac{u_{inter}t_{breakthru} \epsilon}{Z} - \epsilon) / \Lambda`"
+   "Number of mass-transfer units", ":math:`N = \frac{k_{f}a_{s}Z}{u_{bed}}`"
+   "Flow conservation", ":math:`Q_{p, in} - \frac{Q_{bw}t_{bw} + Q_{rinse}t_{rinse}}{t_{cycle}} = Q_{p, out} - \frac{Q_{regen}t_{regen}}{t_{cycle}}`"
+   "Influent total mass of ion", ":math:`M_{in} = Q_{p, in}t_{breakthru}ñ_{in}`"
+   "Removed total mass of ion", ":math:`M_{rem} = V_{bed}q_{eq}n_{op} \rho_{b}`"
+   "Mass of ion in effluent", ":math:`M_{out} = M_{in} - M_{rem}`"
+   "Steady-state effluent concentration (for target ion)", ":math:`ñ_{out} = \frac{M_{out}}{Q_{p, in}t_{breakthru}}`"
+   "Steady-state effluent concentration", ":math:`ñ_{out} = ñ_{in}`"
```

Comment:
Are you looking for \dot{n} here or ñ as a special character?

Reply:
The latter: 
![image](https://user-images.githubusercontent.com/96305519/211892043-89cc78b3-452b-446c-9a4e-5e735f520b76.png)

Reply:
is ñ typically used to express equivalent concentration?

Reply:
I don't know either, I was more curious if it was! If it is the MCAS doc, then it is good with me.